src/main/java/org/LogAggregatorTool/LogAggregatorTool.java

```diff
@@ -0,0 +1,41 @@
+package org.LogAggregatorTool;
+
+import org.LogAggregatorTool.operations.LogFilesExecutor;
+import org.LogAggregatorTool.utility.Constants;
+import java.io.File;
+import java.util.Scanner;
+
+public class LogAggregatorTool {
+
+    public static void main(String[] args){
+        System.out.println(Constants.WELCOME_MESSAGE);
+        Scanner inputScanner = new Scanner(System.in);
+        System.out.println(Constants.NEW_LINE + Constants.ENTER_PATH_MESSAGE);
+        // Taking the user input.
+        String pathToLogFilesFolder=inputScanner.next();
+        File fileObjectOfPathToFolder=new File(pathToLogFilesFolder);
+        // if the path provided by the user is not valid, it iterates until they enter a valid path.
+        while(!fileObjectOfPathToFolder.exists()){
+            System.out.println(Constants.PATH_DOES_NOT_EXIST_MESSAGE);
+            pathToLogFilesFolder=inputScanner.next();
+            if(pathToLogFilesFolder.equals("e")){
+                System.out.println(Constants.EXIT_MESSAGE);
+                return;
+            }
+            fileObjectOfPathToFolder=new File(pathToLogFilesFolder);
```

Comment:
Line 16 can be kept inside while loop to avoid duplicate initialization of fileObjectOfPathToFolder